pkg/api/pod/util_test.go

```diff
@@ -2569,8 +2569,8 @@ func TestDropInPlacePodVerticalScaling(t *testing.T) {
 							Limits:   api.ResourceList{api.ResourceCPU: resource.MustParse("200m")},
 						},
 						ResizePolicy: []api.ContainerResizePolicy{
-							{ResourceName: api.ResourceCPU, RestartPolicy: api.NotRequired},
-							{ResourceName: api.ResourceMemory, RestartPolicy: api.RestartContainer},
+							{ResourceName: api.ResourceCPU, RestartPolicy: api.ResizeRestartPolicyPreferNoRestart},
```

Comment:
Shouldn't all these test cases be testing both?

Reply:
Eh, the value isn't relevant for any of this code. It just checks that it's not nil. I mixed up which value is used, but I don't think it's worth duplicating these.